src/odemis/dataio/test/multi_tiff_test.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+'''
+Created on 23 Jan 2024
+
+@author: Patrick Cleeve
+
+Copyright © 2024, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms 
+of the GNU General Public License version 2 as published by the Free Software 
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; 
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR 
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with 
+Odemis. If not, see http://www.gnu.org/licenses/.
+'''
+import glob
+import json
+import os
+import unittest
+
+import numpy
+
+from odemis import model
+from odemis.dataio import tiff
+
+
+"""This test case requires additional test data to be downloaded from Drive/Software Engineering/Test data/metor/test_images 
+and the associated metadata to be generated using the generate_test_metadata.py script.
+
+To generate the test metadata, run the following command from the root of the repository:
+
+python -m odemis.dataio.test.generate_test_metadata --path <path to test data>
```

Comment:
That `<path to test data>` is hard-coded to `~/test_data/meteor/test_images`, so let's explicitly write that.